AAV.conda_env.yml

```diff
@@ -2,16 +2,19 @@ name: AAV.env
 channels:
     - conda-forge
     - bioconda
-    - anaconda
-    - r
+#   - anaconda
+#   - r
```

Comment:
nit : why leave these in commented at all ? if they arent in use just remove them, the source control has the history

Reply:
Fixed, thanks